### PR TITLE
Add new Couchbase usecase for 0.5

### LINKED ISSUE #896: Create Couchbase Server Usecase
## Is this a BUG REPORT or FEATURE REQUEST
 
 Choose one: FEATURE REQUEST 

**What happened**: New 0.5 feature to be added in user documentation.

### PR DESCRIPTION
**Pull request by mahebbar**
**What this PR does / why we need it**: New usecase for 0.5

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #896

### CODE REVIEW

File: documentation/source/Usecases/couchbase.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    ubuntu@kubemaster:~$ kubectl get pvc
+    NAME                         STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+    couchbase-data-couchbase-0   Bound     pvc-16210b06-c7ba-11e7-892e-000c29119159   5G         RWO            openebs-standard   11h
+    couchbase-data-couchbase-1   Bound     pvc-40e1b64f-c7ba-11e7-892e-000c29119159   5G         RWO            openebs-standard   11h
+
+Get the status of the services using the following command.
+::
+
+    ubuntu@kubemaster:~kubectl get svc
+    ME                                                TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
```
Comment (yudaykiran) [inline documentation/source/Usecases/couchbase.rst]: ME=NAME

Comment (mahebbar) [reply documentation/source/Usecases/couchbase.rst]: Done!

Comment (mahebbar) [reply documentation/source/Usecases/couchbase.rst]: Not including screenshots as the steps/procedure is obvious to the user and easy to follow. We can discuss if it is required.

**Comment by kmova** [comment]
@mahebbar -- The changes related to Couchbase are updated as part of the PR https://github.com/openebs/openebs/pull/906. 

Closing this PR.

**Pull request closed without merging**